### DISCUSSION TITLE
Mistral.rs (inside Docker?) inside CapRover on Mac

### DISCUSSION THREAD

**Discussion by andrebaerlocher** [EricLBuehler/mistral.rs#872] [Q&A]
Hiya, struggling Rust-, Docker- and CapRover-Noob here

I tried to run mistral.rs inside CapRover, in order to access it via the internet. I messed up, it didn't work. I tried to learn about it and stumbled upon some infos: Apparently, [Docker doesn't really work with Apple Silicon](https://chariotsolutions.com/blog/post/apple-silicon-gpus-docker-and-ollama-pick-two/). I tried several other methods that - in hindsight - were stupid. 

So, anyone willing and able help me?

Basically, I have access to a M2 Ultra which I'd like to run mistral.rs. I'd like to access it via API via the internet. I have CapRover running (if that's relevant).
